python_docs_theme/static/pydoctheme_dark.css

```diff
@@ -1,178 +1,332 @@
-/* Common colours */
+/* Updated Common colors */
```

Comment:
This change isn't needed
```suggestion
/* Common colours*/
```